sns/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "sns_arn" {
```

Comment:
Should this output name be more specific to log aggregation> Just thinking if further sns topics are added in the future